src/libcollections/vec.rs

```diff
@@ -1633,6 +1634,15 @@ impl<T> DoubleEndedIterator for IntoIter<T> {
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<T> ExactSizeIterator for IntoIter<T> {}
 
+#[stable(feature = "vec_into_iter_clone", since = "1.7.0")]
```

Comment:
I don't know, you tell me. I looked at the current release at the home page, it says 1.6.0.